### DISCUSSION TITLE
outputs append to one file

### DISCUSSION THREAD

**Discussion by Mr-xn** [projectdiscovery/httpx#618] [Q&A] [answered]
### Please describe your feature request:

perhaps need a probes  t0 make outputs append to one file

**Comment by ehsandeep** [comment, marked as answer]
`echo hackerone.com | httpx | tee -a output.txt` should be sufficient @Mr-xn? in this way, you can have both types of output files if you want, httpx output flag to write to file and `tee -a` write all output to the same file?

**Reply by Mr-xn** [reply]
yeh, I've do this already,However, I still think  a append parameter will be better used in some cases.
